README.md

```diff
@@ -16,6 +16,7 @@ ActiveAdmin Addons will extend your ActiveAdmin and enable a set of addons you c
   - [List](#list): Show Arrays or Hashes like a list.
   - [Color Picker](#color-picker): Select colors from a pretty popup.
   - [Date Time Picker](#date-time-picker): Pick date and time comfortably.
+  - [Date Time Picker Filter](#date-time-picker-filter): Filter your results using a date range. Using [Date Time Picker](#date-time-picker).
```

Comment:
@joseglego The link does not work. `#date-time-picker-filter` needs to match the section title.